ur_bringup/launch/ur_moveit.launch.py

```diff
@@ -249,6 +260,11 @@ def generate_launch_description():
     kinematics_yaml = load_yaml("ur_moveit_config", "config/kinematics.yaml")
     robot_description_kinematics = {"robot_description_kinematics": kinematics_yaml}
 
+    joint_limits_yaml = load_yaml(
```

Comment:
Similar to what gavanderhoorn said, I would prefer not to duplicate the joint_limits.yaml that was loaded at L160. Can you re-use that parameter?

Reply:
I can, and will do this in alternative way. I'll ping you when it is finished

Reply:
7c862d6